content/docs/usage/setup/access.mdx

```diff
@@ -2,4 +2,51 @@
 title: Getting Access
 ---
 
-Coming Soon.
+## T6 Hardware
+### With a monitor and keyboard available
+
+- Insert the microSD card with image flashed.
+- Ensure you have an ethernet cable plugged into your T6 box.
+- Plug in a HDMI screen, make sure it's plugged into HDMI1.
+- Plug in a keyboard.
+- Plug power in.
+- Wait until the box finishes booting, this could take anywhere from 2 to 10 minutes.
+- Once finished, login with the username shibe and the password suchpass.
+- Enter command ip addr and take note of your IP address.
```

Comment:
```suggestion
- Enter command `ip addr` and take note of your IP address.
```

Reply:
We should probably add a `dbx ip` command that wraps everything and returns just your IP as an ease of use thing?